src/test/unit/controller/helpers/TribunalOrderOrRequestHelper.test.ts

```diff
@@ -208,7 +206,200 @@ describe('Tribunal order or request helper', () => {
 
     activateTribunalOrdersAndRequestsLink([notificationWithOrder], request);
     expect(request.session.userCase.hubLinksStatuses[HubLinkNames.TribunalOrders]).toStrictEqual(
-      HubLinkStatus.IN_PROGRESS
+      HubLinkStatus.NOT_STARTED_YET
+    );
+  });
+
+  it('tribunal orders and requests section should be not viewed when not viewed and a response is not required', () => {
+    const request = mockRequest({});
```

Comment:
as I've commented above, if we are only passing userCase to the helper function then these tests can probably be simplified and easier to be read